### PR TITLE
Use richtext for image captions [SF3-2]

### PR DESCRIPTION
**Pull request by lowlandjuju**
Fixes SF3-2

Replaces CharBlock with RichTextBlock,

### CODE REVIEW

File: logbooks/templates/logbooks/story_blocks/image.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {% load wagtailimages_tags %}
 {% image value.image min-680x400 class='img-fluid' %}
-<p class='fs-7 text-grey mt-3 mb-0'>{{ value.caption }}</p>
\ No newline at end of file
+<div class='fs-7 text-grey mt-3 mb-0'>{{ value.caption }}</div>
```
Comment (janbaykara) [inline logbooks/templates/logbooks/story_blocks/image.html:3]: Ah, one extra thing here. We'll need to use the `richtext` filter to render `value.caption` correctly. [Docs for that here.](https://docs.wagtail.org/en/stable/topics/writing_templates.html#rich-text-filter)

**Review by janbaykara — CHANGES_REQUESTED** [review]

File: logbooks/models/blocks.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ class Meta:
 
 class ImageBlock(blocks.StructBlock):
     image = ImageChooserBlock(required=False)
-    caption = blocks.CharBlock()
+    caption = blocks.RichTextBlock()
```
Comment (janbaykara) [inline logbooks/models/blocks.py:25]: We might want to limit the features an editor can add to the rich text. Probably links and basic text formatting but, for example, no embedding images or anything. [Docs on that here.](https://docs.wagtail.org/en/stable/topics/writing_templates.html#rich-text-filter)

**Review by janbaykara — CHANGES_REQUESTED** [review]

**Comment by janbaykara** [comment]
@lowlandjuju bumping this, just need to add `features=[]` to the Rich Text

**Comment by lowlandjuju** [comment]
Thanks @janbaykara - latest commit addresses your two comments above :point_up: x

**Pull request merged**